src/validation/validationRules.js

```diff
@@ -216,6 +216,10 @@ var validations = {
         }
         return false
     },
+    atLeastOneMainCategory(values, value) {
+        return CONSTANT.MAIN_CATEGORY_MAPPING
```

Comment:
I believe here we should use the fetched keyword set for validation, not the hardcoded list.